brianlong/www.validators.app
# 182235395 user watchlist

#### What's this PR do?
- users should be able to add validators to watchlist

#### How should this be manually tested?
- log in
- you should be able to see the eye under the avatars on the main page - it adds validator to watchlist
- to filter by watched validators click the eye on the top of the table (over the avatars column)
- run tests

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/182235395)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [Y] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
